shopno/index.html

```diff
@@ -0,0 +1,586 @@
+                                        <img class="social1__icon" src="./src/images/team/team-mail.png" alt="Social icon">
+                                    </a>
+                                </li>
+                            </ul>
+                        </div>
+                    </li>
+                    <li class="team-list__item">
+                        <picture class="team-list__picture">
+                            <source
+                                srcset="./src/images/team/team-1.webp"
+                                type="image/webp"
+                            >
+                            <img loading="lazy" class="team-list__image" src="./src/images/team/team-1.jpg" width="370" height="440" alt="Team chamber">
+                        </picture>
+                        <div class="team-list__info">
+                            <h3 class="team-list__title">
+                                AL RAYHAN <span class="team-list__title-addition">/ UI Designer</span>
+                            </h3>
+                            <p class="team-list__text">
+                                Lorem Ipsum is not simply is  an action designer random text
+It has roots in a piece. 
+                            </p>
+                            <ul class="team-list__social social1">
+                                <li class="social1__item">
+                                    <a href="#" class="social1__link">
+                                        <img class="social1__icon" src="./src/images/team/team-facebook.png" alt="Social icon">
+                                    </a>
+                                </li>
+                                <li class="social1__item">
+                                    <a href="#" class="social1__link">
+                                        <img class="social1__icon" src="./src/images/team/team-twitet.png" alt="Social icon">
+                                    </a>
+                                </li>
+                                <li class="social1__item">
+                                    <a href="#" class="social1__link">
+                                        <img class="social1__icon" src="./src/images/team/team-ball.png" alt="Social icon">
+                                    </a>
+                                </li>
+                                <li class="social1__item">
+                                    <a href="#" class="social1__link">
+                                        <img class="social1__icon" src="./src/images/team/team-mail.png" alt="Social icon">
+                                    </a>
+                                </li>
+                            </ul>
+                        </div>
+                    </li>
+                </ul>
+            </div>
+        </section>
+        <section class="contact">
+            <h2 style="display: none;">Our contact</h2>
```

Comment:
It's not recommended to use inline styles.

Reply:
```suggestion
            <h2 class="hidden">Our contact</h2>
```